puppet/files/mysql/my.cnf

```diff
@@ -106,7 +106,10 @@ max_binlog_size         = 100M
 # ssl-ca=/etc/mysql/cacert.pem
 # ssl-cert=/etc/mysql/server-cert.pem
 # ssl-key=/etc/mysql/server-key.pem
-
+#
+# * SQL mode
+#
+sql_mode="STRICT_ALL_TABLES"
```

Comment:
It isn't on heroku at least. But it should be! We need some way to make sure we have the same MySQL configuration everywhere.